backend/app/literature/crud/file_crud.py

```diff
@@ -25,7 +25,7 @@
 def create(db: Session, s3: BaseClient, parent_entity_type : str, curie: str, file_contents: str, display_name: str, content_type: str) -> str:
     filename, file_extension = os.path.splitext(display_name)
     bucket_name = 'agr-literature'
-    md5sum = hashlib.md5(file_contents).hexdigest()
+    md5sum = hashlib.md5(bytes(file_contents, "utf-8")).hexdigest()
```

Comment:
I am not sure if all the files will be utf-8. some

Reply:
Yeah that's a good point. I wonder if there's a different mode other than utf-8 that would be better suited? Since the files are coming in as text (str) I assumed they'd just be text based files. If we can upload things like images too maybe the `file_contents` should actually be `bytes` or `bytearray` straight away?

Reply:
I am not sure, we will have to test with different types of files. I am pretty sure I was testing with mainly random images on my computer initially. If the syntax was converting it to a string array then we need to fix it, even if it was working as is.